src/sortStudents.ts

```diff
@@ -1,16 +1,67 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function averageInArray(person: Student): number {
```

Comment:
Use verbs in function naming
```suggestion
function getAverageGrades(person: Student): number {
```